kubernetes/samples/charts/voyager/README.md

```diff
@@ -118,18 +118,18 @@ $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:30305/testwebapp/
 $ curl -k -H 'host: domain2.org' https://${HOSTNAME}:30307/testwebapp/
 ```
 
-## SSL Termination at Ingress Controller
-This sample demonstrates how to terminate SSL trafik at ingress contoller to access WebLogic console thru SSL port. 
+## SSL termination at ingress controller
+This sample demonstrates how to terminate SSL traffic at ingress controller to access the WebLogic Server Administration Console through SSL port. 
 
-### 1. Enable "WebLogic Plugin Enabled" on WebLogic Domain level
+### 1. Enable "WebLogic Plugin Enabled" on the WebLogic domain level
 
-If you are using wdt to configure WebLogic Domain, you need to add the following resource section at the domain level to model yaml file.
+If you are using WDT to configure the WebLogic domain, you need to add the following resource section at the domain level to model YAML file.
 ```
 resources:
      WebAppContainer:
          WeblogicPluginEnabled: true
 ```
-If you are using WLST script to configure domain, following modification needed to respective py script.
+If you are using WLST script to configure the domain, then the following modificationis are needed to respective PY script.
```

Comment:
using WLST script -> using a WLST script
modificationis (typo) -> modifications
to respective -> to the respective

Reply:
done